gto/index.py

```diff
@@ -126,7 +130,15 @@ def write_state(self, path_or_file: Union[str, IO]):
 
     @not_frozen
     def add(
-        self, name, type, path, must_exist, labels, description, update
+        self,
+        name,
+        type,
+        path,
+        must_exist,
+        labels,
+        description,
+        update,
+        requirements=None,
```

Comment:
Tried to add it in backward-compatible way to don't break existing calls, not sure if this methods are "public"